### ISSUE TITLE
WCAG 2.2 upgrade evaluation for VA.gov standards 

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#67023]
## About your team

- Team name:  VA.gov (this will affect all VFS teams, Design System, Governance, etc. )
-	OCTO-DE product owners: @artsymartha68, @humancompanion-usds, @DanielleThierryUSDSVA 
- Governance product owner: @shiragoodman 
-	Product manager: 
-	Designer: 
- Accessibility specialists: @briandeconinck @rsmithadhoc @laflannery 
- Content writer: 
-	[Slack channel discussion](https://dsva.slack.com/archives/C023ZKDJJK1/p1696514093490149)
 
### Describe what you need in a few sentences:

WCAG 2.2 became official today. It would be great to update VA.gov to these standards. But what does that actually mean? As the mobile team already discovered, there may be a relatively small impact to our current processes and standards. 

Thinking the analysis could possibly be done first in a quarter timeframe, then rollout and updating all the affected processes would have to be planned for a longer timeframe with coordination across many teams, including Governance, Design System, Design, etc. 

- Some months ago, Tiffany Pender from the former Shared Services team did [this analysis](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/projects/wcag-2.2/va.gov-and-wcag-2.2.md)
- There are many resources on the web now too, [this from Level Access](https://www.levelaccess.com/blog/wcag-22-aa-summary-and-checklist-for-website-owners/)
- The native mobile team has already upgraded to 2.2, [their documentation and tickets may be helpful](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/4228)
- @briandeconinck on the Governance team has [a ticket to update governance processes](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49681).
- [Epic ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/20806) for when we did the upgrade to WCAG 2.1 a couple of years ago

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [x] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** ....
- **Provide date if available**: No date yet. 

#### Timing for your next step: 
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [x] Next quarter
  - [x] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** xx/xx/xxxx
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: no collab cycle impact (other than updates to the Governance processes and documentation, of course)
- **Collab. Cycle Ticket Opened (date)**: xx/xx/xxxx
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**:
- **User flows**:
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [x] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.
- [ ] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+jilladams%2C+RLHecht&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by eli-oat** [comment]
As a starting point for this work, [there is an existing document that will be useful](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/projects/wcag-2.2/va.gov-and-wcag-2.2.md).

Likewise, the Flagship Mobile app team has done a lot of work around this that will be useful (this mostly lives in google right now, but I now how to get to it when ready).

**Comment by sara-amanda** [comment]
@eli-oat can you link to this here or in Slack? 

> Likewise, the Flagship Mobile app team has done a lot of work around this that will be useful (this mostly lives in google right now, but I now how to get to it when ready).

cc: @coforma-terry

**Comment by eli-oat** [comment]
@artsymartha68 this work is done, [but waiting on review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/CAIA/migrating-to-wcag-2.2.md) -- feel free to take a look and let me know if there is any more information you'd like the document to include.

**Comment by coforma-jamie** [comment]
Brian and Martha have reviewed this guidance. Brian used it as a basis for his writeup of Collab Cycle WCAG 2.2 enforcement, which launched on March 13. This ticket should've been closed awhile ago, closing it now :)
